### PR TITLE
[Topic Entry] Statistics

### PR DESCRIPTION
**Pull request by Sancerro**
### Description

-Added statistics under general.
-Closes issue #2949 
### Type of Change

- Adding a new entry
- Adding a new image

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by SSwiniarski** [comment]
@Sancerro not a problem. You're right it should be short and have terms underneath it. I just thought another paragraph or so would be appropriate. As for the open source content, I understand, but most content like that requires derivative work to have the same sort of license in order to use it, and Docs isn't operating on that type of license.

**Review by SSwiniarski — APPROVED** [review]
@Sancerro I fixed a problem where you modified the `yarn.lock` file, which shouldn't be touched, and now it looks good for a second review.

Also a note for future PRs: You shouldn't be making commits to your main branch, you should make a separate branch off of main and make your changes there. Using main is problematic if you ever intend to have more than one PR open at a time.

**Comment by Sancerro** [comment]
> @Sancerro I fixed a problem where you modified the `yarn.lock` file, which shouldn't be touched, and now it looks good for a second review.
> 
> Also a note for future PRs: You shouldn't be making commits to your main branch, you should make a separate branch off of main and make your changes there. Using main is problematic if you ever intend to have more than one PR open at a time.

Just making sure I'm just supposed to wait now right?

**Comment by SSwiniarski** [comment]
> > @Sancerro I fixed a problem where you modified the `yarn.lock` file, which shouldn't be touched, and now it looks good for a second review.
> > Also a note for future PRs: You shouldn't be making commits to your main branch, you should make a separate branch off of main and make your changes there. Using main is problematic if you ever intend to have more than one PR open at a time.
> 
> Just making sure I'm just supposed to wait now right?

@Sancerro yes, another maintainer will be along with another review.

File: content/general/concepts/statistics/statistics.md
Comment (CBID2) [inline content/general/concepts/statistics/statistics.md:14]:
```suggestion
Statistics is used extensively in fields such as computer science, economics, sociology, physics, and many more. It is also a core component of **machine learning**, a subset of artificial intelligence that uses algorithms to perform specific tasks without explicit instructions. Machine learning algorithms require statistical methods to extract significant patterns, relationships, and insights from complex datasets.

```

File: content/general/concepts/statistics/statistics.md
Comment (CBID2) [inline content/general/concepts/statistics/statistics.md:17]:
```suggestion
**Python** is the most popular programming language for data science. It is used for data analysis, data visualization, machine learning, and many more. Python is a great choice for beginners because it is easy to learn and has a wide range of applications. This programming language is also great for libraries such as **NumPy**, **Pandas**, **Matplotlib**, and **Scikit-learn**.
```

File: content/general/concepts/statistics/statistics.md
Comment (CBID2) [inline content/general/concepts/statistics/statistics.md:24]: [question]: How are these libraries applicable to statistics? I think adding a brief explanation about this topic would help developers correlate this list back to your entry's topic.

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Sancerro. I made some suggestions on how to improve your entry. In the future, consider using Grammarly to help you revise them.

**Comment by Sancerro** [comment]
Thank you for your time!

**Review by yangc95 — APPROVED** [review]

**Pull request merged**
